src/ui/public/styles/base.less

```diff
@@ -3,6 +3,9 @@
 @import (reference) "./variables";
 @import (reference) "~ui/styles/bootstrap/bootstrap";
 
+// Kibana UI Framework
+@import (less) "../../../../node_modules/@elastic/kibana-ui-framework/dist/framework.css";
```

Comment:
the `~` is what points the require statements to use webpack resolution, so this should be:

```
@import (less) "~@elastic/kibana-ui-framework/dist/framework.css";
```